src/version.js

```diff
@@ -1 +1 @@
-export default '1.1.0';
+export default '1.2.0-beta';
```

Comment:
Lets do `3.0.0-beta`

Reply:
do we need an intermediate v3.0.0-beta.1 branch, or can this go directly to master?

Reply:
It can go to master